extension/healthcheckextension/healthcheckextension_test.go

```diff
@@ -135,6 +176,71 @@ func TestHealthCheckExtensionUsageWithCheckCollectorPipeline(t *testing.T) {
 	require.Equal(t, http.StatusInternalServerError, resp3.StatusCode)
 }
 
+func TestHealthCheckExtensionUsageWithCustomPathWithCheckCollectorPipeline(t *testing.T) {
+	config := Config{
+		TCPAddr: confignet.TCPAddr{
+			Endpoint: testutil.GetAvailableLocalAddress(t),
+		},
+		CheckCollectorPipeline: checkCollectorPipelineSettings{
+			Enabled:                  true,
+			Interval:                 "5m",
+			ExporterFailureThreshold: 1,
+		},
+		Path: "/health",
+	}
+
+	hcExt := newServer(config, zap.NewNop())
+	require.NotNil(t, hcExt)
+
+	require.NoError(t, hcExt.Start(context.Background(), componenttest.NewNopHost()))
+	t.Cleanup(func() { require.NoError(t, hcExt.Shutdown(context.Background())) })
+
+	// Give a chance for the server goroutine to run.
+	runtime.Gosched()
+
+	newView := view.View{Name: exporterFailureView}
+
+	currentTime := time.Now()
+	vd1 := &view.Data{
+		View:  &newView,
+		Start: currentTime.Add(-2 * time.Minute),
+		End:   currentTime,
+		Rows:  nil,
+	}
+	vd2 := &view.Data{
+		View:  &newView,
+		Start: currentTime.Add(-1 * time.Minute),
+		End:   currentTime,
+		Rows:  nil,
+	}
+
+	client := &http.Client{}
+	url := "http://" + config.TCPAddr.Endpoint + config.Path
+	resp0, err := client.Get(url)
+	require.NoError(t, err)
+	defer resp0.Body.Close()
```

Comment:
Can I suggest the following:
```suggestion
require.NoError(t, resp0.Body.Close(), "Must be able to close the response")
```

I suspect the linter may highlight these are bad nobody closed (at least in future if not now)

Reply:
done